ecl/baremetal/v2/server.py

```diff
@@ -155,9 +155,9 @@ def start(self, session, server_id, boot_mode):
         self._translate_response(resp, has_body=False)
         return self
 
-    def stop(self, session, server_id):
+    def stop(self, session, server_id, type):
         uri = self.base_path % server_id
-        body = {"os-stop": None}
+        body = {"os-stop": {"type": type}}
```

Comment:
should request `{"os-stop": None}` if type argument is not passed because most regions don't have new baremetal API.